src/ucp/api/ucp.h

```diff
@@ -262,7 +262,8 @@ enum {
                                             address for allocation. */
     UCP_MEM_MAP_FIXED    = UCS_BIT(2)  /**< Don't interpret address as a hint:
                                             place the mapping at exactly that
-                                            address. */
+                                            address. Address must be a multiple
```

Comment:
@evgeny-leksikov  once you make the chance consider that it is auto-approved by me :) but you have to get approval from @bbenton as well.